src/applications/mhv-secure-messaging/containers/AuthorizedRoutes.jsx

```diff
@@ -28,6 +30,9 @@ AppRoute.propTypes = {
 };
 
 const AuthorizedRoutes = () => {
+  const contactListPage = useSelector(
+    state => state.featureToggles.mhvSecureMessagingEditContactList,
+  );

```

Comment:
I think you may want to use this approach to look up the feature toggle value
```suggestion
 const contactListPage = useSelector(
    state =>
      state.featureToggles[
        FEATURE_FLAG_NAMES.mhvSecureMessagingEditContactList
      ],
  );

```